core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -104,6 +127,7 @@ public String toString() {
       buffer.append(" (").append(deadlineNanoTime - System.nanoTime()).append(" ns from now)");
     }
     buffer.append("]");
+    buffer.append("\ncodec=").append(codec);
```

Comment:
I was thinking the [] was supposed to enclose all the options, so that we would have `[deadlineNanoTime=12341234 (50 ns from now), codec=SOMETHING]`